README.md

```diff
@@ -29,6 +29,18 @@ module.exports = function (items) {
 }
 ```
 
+Or create an element without using template literals:
+
+```js
+// list.js
+var bel = require('bel/create-element')
+
+module.exports = function (items) {
+  return bel('ul', items.map(item => bel('li', item)))
```

Comment:
@chromakode yeah, I think that would be preferable; `bel` + `yo-yoify` have mad browser compatibility - introducing fat arrow functions would mess with that